hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieParquetInputFormat.java

```diff
@@ -209,6 +210,12 @@ public Configuration getConf() {
             colNamesWithTypesForExternal.size(),
             true);
       }
+    } else if (split instanceof RealtimeSplit) {
```

Comment:
can we do this at the sub class? Not sure if we want to make this class aware of RealtimeSplit